app/models/concerns/indexable.rb

```diff
@@ -4,6 +4,7 @@ module Indexable
   included do
     include Elasticsearch::Model
     index_name SearchIndex.index_name(self)
+    attr_accessor :relevancy
```

Comment:
Yes exactly, it's just an attribute that will be set after we get the ES results and before we pass the objects to the presenters.